src/client.ts

```diff
@@ -1985,7 +1988,9 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
       return channel;
     }
     const channel = new Channel<StreamChatGenerics>(this, channelType, channelID, custom);
-    this.activeChannels[channel.cid] = channel;
+    if (!this.options.disableCache) {
```

Comment:
Could this be necessary also in `getChannelByMembers` method?

Reply:
We do ! Thanks, missed that one. 

We're also missing it in `channel.query` for adding the config. Adding them in a sec.